# Service to train a singulation detection model
string logFolderName       #Folder where to look for log files to train the model.
string modelName           #Name of the model to save (without any extension).
---
float64 error	           #Cross validation error in training.
int64 ret
string msg